gson/src/main/java/com/google/gson/internal/ReflectionAccessFilterHelper.java

```diff
@@ -98,6 +98,7 @@ private static abstract class AccessChecker {
             }
           };
         } catch (NoSuchMethodException ignored) {
+          // Ignored exception
```

Comment:
I usually use something like `// OK: <reason why it is OK>`. Here, maybe:

`// OK: will assume everything is accessible`

Though I'm not sure it really _is_ OK. I think it might be better to wrap the exception in an `IllegalStateException` or something. But if we do that it should be in a separate PR.